app/repositories/application/GeneralApplicationRepository.scala

```diff
@@ -493,6 +477,24 @@ class GeneralApplicationMongoRepository(timeZoneService: TimeZoneService,
     reportQueryWithProjectionsBSON[ApplicationForDiversityReport](query, projection)
   }
 
+  override def onlineTestPassMarkReport(frameworkId: String): Future[List[ApplicationForOnlineTestPassMarkReport]] = {
+    val query = BSONDocument("$and" -> BSONArray(
+      BSONDocument("frameworkId" -> frameworkId),
+      BSONDocument(s"progress-status.PHASE1_TESTS_RESULTS_RECEIVED" -> true)
+    ))
+
+    val projection = BSONDocument(
+      "userId" -> "1",
+      "applicationId" -> "1",
+      "scheme-preferences.schemes" -> "1",
+      "assistance-details" -> "1",
+      "testGroups" -> "1",
+      "progress-status" -> "2"
```

Comment:
I change it to 1,we need it in the projection. I do not know what 2 means, it does not seem to be documented anywhere, we are using that in the other reports.